docs-devsite/vertexai.schemashared.md

```diff
@@ -0,0 +1,103 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# SchemaShared interface
+Basic [Schema](./vertexai.schema.md#schema_class) properties shared across several Schema-related types.
+
+<b>Signature:</b>
+
+```typescript
+export interface SchemaShared<T> 
+```
+
+## Properties
+
+|  Property | Type | Description |
+|  --- | --- | --- |
+|  [description](./vertexai.schemashared.md#schemashareddescription) | string | Optional. The description of the property. |
+|  [enum](./vertexai.schemashared.md#schemasharedenum) | string\[\] | Optional. The enum of the property. |
+|  [example](./vertexai.schemashared.md#schemasharedexample) | unknown | Optional. The example of the property. |
+|  [format](./vertexai.schemashared.md#schemasharedformat) | string | Optional. The format of the property. |
+|  [items](./vertexai.schemashared.md#schemashareditems) | T | Optional. The items of the property. |
+|  [nullable](./vertexai.schemashared.md#schemasharednullable) | boolean | Optional. Whether the property is nullable. |
+|  [properties](./vertexai.schemashared.md#schemasharedproperties) | { \[k: string\]: T; } | Optional. Map of Schemas. |
```

Comment:
NOT BLOCKING

This should _probably_ be this?

Map of `Schema` objects.

Reply:
Fixed.

Reply:
I'm still seeing:

Map of Schemas.

instead of:

Map of `Schema` objects.

Reply:
I think I missed a second instance of it. Should be fixed now.